DDSTextureLoader/DDSTextureLoader12.cpp

```diff
@@ -1283,7 +1283,7 @@ namespace
             &defaultHeapProperties,
             D3D12_HEAP_FLAG_NONE,
             &desc,
-            D3D12_RESOURCE_STATE_COPY_DEST,
```

Comment:
The "standalone" copies of DDSTextureLoader, WICTextureLoader don't support Xbox.